indykite/utilities.go

```diff
@@ -409,6 +409,13 @@ var OAuth2ClientSubjectTypes = map[string]configpb.ClientSubjectType{
 // OAuth2ClientSubjectTypesReverse defines all supported Client Subjects and its reversed mapping.
 var OAuth2ClientSubjectTypesReverse = ReverseProtoEnumMap(OAuth2ClientSubjectTypes)
 
+// AuthorizationPolicyStatusTypes defines all supported StatusTypes and its mapping.
+var AuthorizationPolicyStatusTypes = map[string]configpb.AuthorizationPolicyConfig_Status{
+	"STATUS_INVALID":  configpb.AuthorizationPolicyConfig_STATUS_INVALID,
+	"STATUS_ACTIVE":   configpb.AuthorizationPolicyConfig_STATUS_ACTIVE,
+	"STATUS_INACTIVE": configpb.AuthorizationPolicyConfig_STATUS_INACTIVE,
+}
```

Comment:
Why you have the key `"STATUS_ACTIVE"` so into Terraform file I have to write:

```tf
status = "STATUS_ACTIVE"
```

Instead of 

```tf
status = "active"
```

Reply:
For consistency with our proto definition, but i can change that.
`AuthorizationPolicyConfig_Status_name = map[int32]string{
	0: "STATUS_INVALID",
	1: "STATUS_ACTIVE",
	2: "STATUS_INACTIVE",
}
AuthorizationPolicyConfig_Status_value = map[string]int32{
	"STATUS_INVALID":  0,
	"STATUS_ACTIVE":   1,
	"STATUS_INACTIVE": 2,
}`